src/components/atoms/HeaderCell/HeaderCell.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './HeaderCell.css';
+
+const HeaderCell = ({ children }: { children: string }) => {
+    return (
+        <th role="rowheader" className="a-header-cell">{children}</th>
```

Comment:
We might need to think about this explicit role here. I think the `<th>` will have an implicit role of either `rowheader` or `columnheader` depending on where it is used in the table. 

If we explicitly define it here, then it becomes a restraint to only be used to head rows.

Reply:
Great catch. I used this to check the table header cells here: https://github.com/codurance/react-interview/pull/2/files#diff-c0e078e0aa0c8a4cad4b0625af03cd9985bf534aeff295a449a18dbb0b28e05dR24
Should definitely make it more robust.